shared/chat/conversation/list-area/index.native.tsx

```diff
@@ -76,6 +77,10 @@ const useScrolling = (p: {
   }
 }
 
+// This keeps the list stable when data changes. If we don't do this it will jump around
+// when new messages come in and its very easy to get this to cause an unstoppable loop of
+// quick janking up and down
+const maintainVisibleContentPosition = {autoscrollToTopThreshold: 1, minIndexForVisible: 0}
```

Comment:
this was removed as it causes all incoming messages to slide in, but its required to solve the much worse problem where it'll permanently flick back and forth.